.github/workflows/ci.yml

```diff
@@ -30,6 +30,9 @@ jobs:
                   tools: composer:v2
                   coverage: none
 
+            - name: Composer Update
+              run: composer update
```

Comment:
We'll also need the caching (and, to keep things consistent to ease maintenance for us, move the PHP version into a matrix). I'll create a pre-patch for this.